### DISCUSSION TITLE
Interfaces not visible

### DISCUSSION THREAD

**Discussion by tkalfaoglu** [dream-hunter/netflow#10] [Q&A] [answered]
Hi there.. many thanks for this program!

I installed it on an ubuntu system, configured the nearby CISCO 3625.. It shows the same device in DEVICES page as two:  one for the loopback IP and one for the local IP..

On the INTERFACES page I see the same two "devices" but neither has any interface under them.. so I cannot see any data.. 

I know the Analyzer and collector are both working, their database is growing
Database schemas size
analyzer  12.0 MB
collector  1.8 MB
dashboard 112.0 kB

What am I missing?
thanks! -t

**Comment by dream-hunter** [comment, marked as answer]
Hello.

You have to retrieve the interface list via SNMP:
1. Configure your cisco router:
```
conf t
snmp-server community yourcommunity RO
exit
wr mem
```
2. Go to web interface -> devices tab, select your route and press edit. Add your SNMP community.
3. Get interface list on interfaces tab.
4. Check needed interfaces and make them active (green).
5. Wait a little bit before information appears.

**Reply by tkalfaoglu** [reply]
Thank you very much! Trying it now :)

**Reply by tkalfaoglu** [reply]
It's been about 20 minutes and they are not populating.. what else shall I try?
thanks

**Reply by dream-hunter** [reply]
Did you retrieved list the interfaces?
Did you activate both devices and interfaces on it?
[Did you configure your routers to send netflow data?](https://github.com/dream-hunter/netflow/wiki/Sensors-configuration#cisco-routers)
What netflow version you applied?

**Reply by tkalfaoglu** [reply]
Many thanks.. for some reason on the DEVICES tab it shows it as two devices: 192.168.1.1 and 10.0.0.1  -- 10 is the Loopback and 192.168.1.1 is Giga 0/1 - which is the local LAN.. Neither has any interfaces listed.. I tried clicking DISCOVER after selecting both "devices"  and shows nothing within. 

Yes, I thought I configured it..  I went with V5.. 
it holds a  pppoe connection to the vdsl modem which runs in bridged mode..
Your software is running on 192.168.1.191..   Both devices are enabled btw..  (there is actually 1 router altogether)
Config below, I changed the outside IP below for privacy.

version 15.4
aaa new-model
aaa local authentication attempts max-fail 5
ip flow-cache timeout active 1
ip domain name kalfaoglu.net
ip inspect tcp reassembly queue length 64
ip inspect name FWOUT tcp
ip inspect name FWOUT udp
ip inspect name FWOUT icmp
ip inspect name FWOUT smtp
ip inspect name FWOUT dns
ip inspect name FWOUT ftp
ip inspect name FWOUT http
ip inspect name FWOUT https
ip inspect name FWOUT imap
ip inspect name FWOUT imap3
ip inspect name FWOUT imaps
ip inspect name FWOUT mysql
ip inspect name FWOUT ntp
ip inspect name FWOUT nfs
ip inspect name FWOUT pop3
ip inspect name FWOUT pop3s
ip inspect name FWOUT ssh
ip inspect name FWOUT time
ip inspect name FWOUT x11
ip cef
login block-for 120 attempts 5 within 60
login on-failure log
login on-success log
no ipv6 cef

interface Loopback0
 ip address 10.0.0.1 255.255.255.0
 ip flow ingress
 ip flow egress
!
interface GigabitEthernet0/0
 description *********** This is towards MODEM!
 bandwidth 27000
 no ip address
 ip access-group INBOUND in
 ip flow ingress
 ip flow egress
 ip virtual-reassembly in
 ip tcp adjust-mss 1452
 duplex auto
 speed auto
 pppoe enable group global
 pppoe-client dial-pool-number 1
 no lldp transmit
 no lldp receive
 no cdp enable
!
interface GigabitEthernet0/1
 description ******* inside network!!
 ip address 192.168.1.1 255.255.255.0
 ip nat inside
 ip virtual-reassembly in
 ip tcp adjust-mss 1452
 duplex auto
 speed auto
 no cdp enable
 no mop enabled
!
interface Dialer1
 mtu 1492 
 bandwidth 27000
 ip address negotiated
 ip access-group INBOUND in
 ip mtu 1452
 ip flow ingress
 ip flow egress
 ip nat outside
 ip inspect FWOUT out
 ip virtual-reassembly in
 encapsulation ppp
 ip tcp adjust-mss 1412
 dialer pool 1
 dialer-group 1
 ppp mtu adaptive
 ppp authentication chap callin
 ppp chap hostname xxxxxxxxxxxxxxxxxx
 ppp chap password 7 xxxxxxxxxxxxxxxx
 ppp ipcp dns request
 no cdp enable
!         
!         
ip forward-protocol nd
!         
no ip http server
ip http access-class 1
ip http authentication local
no ip http secure-server
ip flow-export source Loopback0
ip flow-export version 5
ip flow-export destination 192.168.1.191 9998
!
no ip nat create flow-entries
ip nat inside source list 1 interface Dialer1 overload
ip nat inside source list nat-list interface Dialer1 overload
ip nat inside source static tcp 192.168.1.191 53 192.162.105.101 53 extendable
ip nat inside source static udp 192.168.1.191 53 192.162.105.101 53 extendable
ip default-network 192.162.105.101
ip route 0.0.0.0 0.0.0.0 Dialer1
ip route 192.168.0.0 255.255.0.0 GigabitEthernet0/1
!
ip access-list extended INBOUND
 deny   tcp any any fragments
 deny   udp any any fragments
 deny   icmp any any fragments
 deny   ip any any fragments
 deny   tcp any any eq 1080
 permit udp any any eq domain
 permit tcp any any eq domain
 deny   ip any any
ip access-list extended OUTBOUND
 permit ip any any
!
dialer-list 1 protocol ip permit
!
!
snmp-server community xxxxxxx RO
snmp-server ifindex persist
access-list 1 permit 192.168.1.0 0.0.0.255
access-list 23 permit 192.168.1.0 0.0.0.255
!
control-plane
!
line con 0
 exec-timeout 5 0
line aux 0
line 2
 no activation-character
 no exec
 transport preferred none
 transport output lat pad telnet rlogin lapb-ta mop udptn v120 ssh
 stopbits 1
line vty 0 4
 access-class 23 in
 exec-timeout 60 0
 transport input ssh
line vty 5 15
 access-class 23 in
 transport input all
!
scheduler allocate 20000 1000

**Reply by dream-hunter** [reply]
Check the WiKi's [Web Interface quickstart](https://github.com/dream-hunter/netflow/wiki/Web-Interface-quickstart). I've added it today.
